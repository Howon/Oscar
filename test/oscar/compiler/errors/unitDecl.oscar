def main() => unit = {
  unit a = unit;
  Println(a);
}
