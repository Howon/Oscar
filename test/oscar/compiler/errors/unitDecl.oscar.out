Error: Cannot declare value of type unit to a
